models/engine/db_storage.py

```diff
@@ -76,13 +76,15 @@ def close(self):
         self.__session.remove()
 
     def get(self, cls, id):
+        ''' get obj by cls and id '''
         obj = self.all(cls)
         for key, val in obj.items():
             if id == val.id:
                 return val
         return None
 
     def count(self, cls=None):
+        ''' count number of objs by class '''
```

Comment:
Great code! Comments could use a capital letter at the beginning, and it would make more sense to me to have "val.id == id" instead of the opposite!